### DISCUSSION TITLE
Is there any version specific for integration of jointjs+ to intellij IDE

### DISCUSSION THREAD

**Discussion by RajeshNakkanaboina** [clientIO/joint#2202] [Q&A]
### Introduction

I recently bought a licence for jointjs+, and while connecting it with the Visual Studio Code IDE, everything is going smoothly, but when I try to use the 2020.2v and free-trail versions of the Intellij IDE, nothing works. Exist any integration requirements that are version or ide specific?

### Does your question relate to JointJS or JointJS+. Select both if applicable.

JointJS+

**Comment by jamesgeorgewilliams** [comment]
JointJS is a JavaScript library, so if your IDE supports JavaScript. Everything should work fine. In what sense does "nothing work"?

If you mean coding assistance, such as syntax highlighting, JavaScript support in Intellij seems to be a separate plugin that needs to be enabled. Maybe, JavaScript support is not included in the free trial version.
https://www.jetbrains.com/help/idea/javascript-specific-guidelines.html
